streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -81,8 +80,12 @@
                     new MockStateStoreSupplier(storeName1, false).get(),
                     new MockStateStoreSupplier(storeName2, true).get()
             ),
-            Collections.<String, String>emptyMap(),
-            Collections.<StateStore, ProcessorNode>emptyMap());
+            new HashMap<String, String>() {
+                {
+                    put(storeName1, storeChangelogTopicName1);
+                    put(storeName2, storeChangelogTopicName2);
+                }
+            });
```

Comment:
We are now passing the map from store names to its changlog topics as the last parameter into `ProcessorStateManager`, since we can actually gather that information beforehand in the topology builder already (any stores that are not changelog enabled won't have an entry in the map). So in `ProcessorStateManager` it only needs to access this map to decide whether we need to get a changelog for the state store and what would be the changelog name.

In these tests since we are adding two stores `persistentXX` and `nonPersistentXX` which are all changelogged, we need to pass in this map.